src/components/Sidebar/components/ExpandedList/ExpandedList.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "styled-components/macro";
+
+import ExpandedList from "./ExpandedList";
+
+export default {
+  title: "ExpandedList",
+};
+
+export const ExpandedListStory = () => (
+  <div>
+    <ExpandedList>
+      <a href="#">Hello</a>
```

Comment:
```suggestion
      <a href="/">Hello</a>
```
The eslint is complaining about this so the test dosnt pass. Maybe you can change to "/" to get rid of this error before we add real working href:s. Don´t know if it will work or not.
There are some more # tags across the file.

Reply:
The links were removed in a more recent commit. So this error does not exist any more. Thanks.